engine/pom.xml

```diff
@@ -930,6 +916,18 @@
       <version>${xmlunit.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+      <version>3.1.0</version>
+      <exclusions>
+        <exclusion>
+          <groupId>*</groupId>
```

Comment:
@smmribeiro Why the exclusions?

Reply:
@graimundo, to be honest I don't know/remember why I added those exclusions... even more being a test scoped dependency.
I'll verify it.

Reply:
Resolving as I've updated the code.